packages/devextreme/js/__internal/ui/m_validation_group.ts

```diff
@@ -19,7 +19,7 @@ class ValidationGroup extends DOMComponent {
   _init() {
     // @ts-expect-error
     super._init();
-    ValidationEngine.addGroup(this);
+    ValidationEngine.addGroup(this, false);
```

Comment:
[broken scenario]
Custom ValidationGroup should not be removed when its validators are removed. It was supported [there](https://github.com/DevExpress/DevExtreme/pull/27435/files#diff-64529d0ddad2c54300c7f438acc04e59f59c839d7b324b6ded1f630639c8a014L787). We should not break it.

Reply:
Question - should we cover a custom validationGroup or just simply check validationGroup? Also there are things that I found

1. I think this [line](https://github.com/DevExpress/DevExtreme/pull/27435/files#diff-64529d0ddad2c54300c7f438acc04e59f59c839d7b324b6ded1f630639c8a014L787) seems to not work well with this `ValidationEngine.addGroup(this, false)` code we added. This makes the test evergreen regardless we make validationGroup removable or not.
2. Based from what I understand, validationGroup doesn't support custom validationGroup from [this](https://github.com/DevExpress/DevExtreme/blob/5a2883fa56ff15b64b0c24d79f84b0228c075876/packages/devextreme/js/__internal/ui/m_validation_group.ts#L22). Unlike form where it gets [this](https://github.com/DevExpress/DevExtreme/blob/5a2883fa56ff15b64b0c24d79f84b0228c075876/packages/devextreme/js/__internal/ui/form/m_form.ts#L705), should we rework what validationGroup passes on group? Let me know if I'm missing something out.

Reply:
1. "I think this [line](https://github.com/DevExpress/DevExtreme/pull/27435/files#diff-64529d0ddad2c54300c7f438acc04e59f59c839d7b324b6ded1f630639c8a014L787) seems to not work well with this ValidationEngine.addGroup(this, false) code we added." Yeah, it makes. But it should be removed after we change our check to use `groupConfig._isRemovable`.  I see you already [updated the check](https://github.com/DevExpress/DevExtreme/pull/27435/files#diff-64529d0ddad2c54300c7f438acc04e59f59c839d7b324b6ded1f630639c8a014R787).
2. Didn't get ur question, sorry. The second link does not work.
3. Yeah, that was an initial idea of my note. We should just pass "isRemovable=false" here when we add a groupConfig from ValidationGroup component, and that's should be enough. But during my previous review "ValidationEngine.addGroup(this, false);" meant that this group is removable. That's why i noted